### PR TITLE
Updating arch diagram

### PR DESCRIPTION
**Pull request by jasonmadigan**
Updating the architecture diagram on the getting started guide with one from John.

### CODE REVIEW

**Comment by jasonmadigan** [comment]
@finp FYI

**Comment by finp** [comment]
@jasonmadigan fyi, or just f u?

**Comment by jasonmadigan** [comment]
@finp wanted your take. I think you've got the steps outlined as links below, so might lop them off the diagram and come up with appropriate links. Wondering if links may change between now and next week, though. May be worth holding?

**Comment by finp** [comment]
links will definitely change, altho I don't mind broken links if you wanna change text

**Comment by johnfriz** [comment]
@jasonmadigan @finp - where is this PR at?

**Comment by jasonmadigan** [comment]
2 things left IMHO:

- Crop the arch diagram to yank the sentences
- Move these into links below instead (e.g. move the sentences into actual text below the image)

**Comment by jasonmadigan** [comment]
screen shot 2018-07-02 at 13 10 40

Rebased. Diagram updated with tweaks from @finp

**Comment by jasonmadigan** [comment]
Final resting place for this is to be: https://docs.aerogear.org/aerogear/latest/con_services.html

**Comment by jasonmadigan** [comment]
@finp diagram moved to its final home.

File: modules/ROOT/pages/workflow.adoc [hunk 1]
```diff
@@ -2,10 +2,6 @@
 
 AeroGear Mobile Services provide features described in the section about xref:con_services.adoc[Mobile Services].
 
-The workflow for connecting your mobile app to mobile services can be summarized as:
-
-image::3steps.png[]
-
 . xref:provisioning-services.adoc[Provision and configure a mobile service]
```
Comment (finp) [inline modules/ROOT/pages/workflow.adoc]: these won't work, but can merge and follow up? as long as @johnfriz  is happy with diagram

Comment (johnfriz) [reply modules/ROOT/pages/workflow.adoc]: @finp - can you clarify what you mean by "these won't work"?

Comment (finp) [reply modules/ROOT/pages/workflow.adoc]: not any of the anointed urls, wanna replace everything under the png  with:

The following Mobile Services are available:

* xref:identity-management.adoc[{keycloak-service}]	
* xref:push-notifications.adoc[{unifiedpush-service}]	
* xref:runtime-connector.adoc[{custom-service}]	
* xref:mobile-metrics.adoc[{metrics-service}]	
* xref:mobile-cicd.adoc[{digger-service}]	
* xref:device-security.adoc[{device-security-service}]

Comment (johnfriz) [reply modules/ROOT/pages/workflow.adoc]: @finp - merge away.

**Pull request merged**